org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/SModelUtil.xtend

```diff
@@ -20,7 +20,10 @@ import java.util.function.Consumer
 
 /**
  * Convenient functions for generating sprotty diagrams.
+ * 
+ * @deprecated  
```

Comment:
Maybe add one or two sentences on what is the recommended way of creating diagrams?